packages/validation/src/json/LanguageUtils.ts

```diff
@@ -9,4 +12,45 @@ export class LanguageWrapper extends LionWebJsonChunkWrapper {
     constructor(languageJson: LionWebJsonChunk) {
         super(languageJson);
     }
+    
+    // inheritsFrom = (classifier: LionWebJsonNode): LionWebJsonNode[] => {
```

Comment:
Would switching on https://github.com/cartant/eslint-plugin-etc/blob/main/docs/rules/no-commented-out-code.md help to detect commented-out code?

Reply:
Maybe, but just searching for // will also be quick.

Reply:
LanguageWrapper is unfinished work and LionwebDiff is very outdated.  So we can either leave it as is, or remove them completely. What do you think?

Reply:
I'm very much in favor of removing unfinished work(/"stuff").